src/app/time-entry/time-entry.component.ts

```diff
@@ -66,9 +66,10 @@ export class TimeEntryComponent implements OnInit {
     };
 
     this.entry.addEntry(work).subscribe((result) => {
-      console.log("result", result);
-      this.workForm.reset({});
-      this.totalTime = ["00", "00"];
+      // console.log("result", result);
+      // this.workForm.reset({});
+      // this.totalTime = ["00", "00"];
+      this.entry.newEntry.next(result);
```

Comment:
this line should be part of addEntry function in service 
this.entry.newEntry.next(result);